packages/compat/src/compat-adapters/@glimmer/tracking.ts

```diff
@@ -20,7 +20,7 @@ class RedirectToEmber extends Plugin {
       outputFileSync(join(this.outputPath, 'index.js'), `export { tracked } from '@ember/-internals/metal';`);
       outputFileSync(
         join(this.outputPath, 'primitives', 'cache.js'),
-        `export { createCache, getValue, isConst } from "@ember/-internals/metal";`
+        `export { cached, createCache, getValue, isConst } from "@ember/-internals/metal";`
```

Comment:
@ef4 does this need to be in a macroCondition somehow? since pre ember-source-4.1, this cached export doesn't exist?

Reply:
I think you can write this in a way that isn't a SyntaxError even when `cached` is not exported by ember metal:

```js
import * as metal from "@ember/-internals/metal";
const { cached, createCache, getValue, isConst } = metal;
export { cached, createCache, getValue, isConst };
```

It means our module always exports the name `cached`, but on older ember its value will be undefined.

Reply:
will that work pre 3.27? I know _some_ modules weren't allowing `import *`
I mean, I guess I'll find out!